src/Http/Routing/perf/Microsoft.AspNetCore.Routing.Performance.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Lot of BOM changes that don't change any content.

Reply:
I wish `git rebase --whitespace=fix` corrected this. Do you know a good way to fix it?